requirements_pt.txt

```diff
@@ -1,8 +1,8 @@
 numba
-numpy < 2.0
-peft
```

Comment:
Would removing `peft` affect this functionality? https://github.com/intel/neural-compressor/blob/61f1e393b6374703b6516fe9406bafb0cc088009/neural_compressor/torch/algorithms/smooth_quant/utility.py#L2202

Reply:
The peft installation requires transformers installation, it's a high-level package and shouldn't be included in base requirement. So we move it to example requirement.